cli/tests/testdata/package_json/invalid_value/error.ts.out

```diff
@@ -0,0 +1,6 @@
+error: Parsing version constraints in the application-level package.json is more strict at the moment.
+
+Invalid npm specifier version requirement. Unexpected character.
+  invalid stuff that won't parse
+  ~
+    at file:///[WILDCARD]/error.ts:2:23
```

Comment:
This got me confused - this string is not present in `error.ts:2:23` - it's present in the related package.json. Any chance this could be improved to point to the appropriate `package.json` file?

Reply:
I'm not sure it's worth the effort mostly because we'll improve this over time to support more version constraints. Also, I don't think it will be so big of a deal for the user to figure out because the user will get a diagnostic at the bare specifier which is referenced in the package.json file. I think this just seems confusing because this test uses the string "invalid stuff that won't parse", but the user will see a version constraint like `1.2 - 1.4` or something like that, which should be more obvious.